stdlib/public/core/DebuggerSupport.swift

```diff
@@ -148,6 +148,8 @@ public enum _DebuggerSupport {
       return count == 1 ? "1 element" : "\(count) elements"
     case .`struct`?, .`enum`?, nil:
       switch value {
+      case let x as String:
+        return x.description
```

Comment:
I was hoping it was as simple as this. Might be worth a comment pointing out that `String.debugDescription` has the unwanted (debugging) side effects of escaping newlines.